test/byContract/IexecERC20/IexecERC20.test.ts

```diff
@@ -64,16 +63,22 @@ describe('ERC20', async () => {
 
     describe('Transfer', () => {
         it('Should transfer tokens', async () => {
-            const transferArgs = [recipient.address, BigNumber.from(value)] as [string, BigNumber];
-            expect(await iexecPocoAsHolder.callStatic.transfer(...transferArgs)).to.be.true;
-            await expect(iexecPocoAsHolder.transfer(...transferArgs))
-                .to.changeTokenBalances(iexecPoco, [holder, recipient], [-value, value])
+            const transferArgs = [recipient.address, value] as [string, number];
+            expect(await iexecPocoAsHolder.transfer.staticCall(...transferArgs)).to.be.true;
+            const tx = iexecPocoAsHolder.transfer(...transferArgs);
```

Comment:
Fixes `HardhatChaiMatchersNonChainableMatcherError: The matcher 'emit' cannot be chained after 'changeTokenBalances'.`